source/Glossary/throttling.md

```diff
@@ -1,37 +1,31 @@
 
 {% anchor h2 %}
-What can cause a deferral?
+What causes a deferral?
 {% endanchor %}
 
--   The receiving server doesn't recognize your IP, and is afraid you're sending spam. *This is why [warming up]({{root_url}}/Classroom/Deliver/warming_up_ips.html) is so important, so they know who you are and that they can trust mail from your IPs.*
+-   The receiving server doesn't recognize your IP, and is afraid you're sending spam. *This is why [warming up]({{root_url}}/Classroom/Deliver/warming_up_ips.html) is so important, so that recipeint mail servers can become used to, and anticipate your messages.*
 
 -   The receiving server doesn't have any open ports to receive email.
 
 -   The recipient's mailbox is full.
 
--   Some recipients have marked your mail as spam, but not enough for the server to block you. Instead, they will refuse to receive more until they see what the rest of your current recipients do.
-
-What happens if it's continuously deferred for 72 hours? Well, that generally means it's an issue like the account being inactive, having a full mailbox, or some other legitimate problem with the recipient. When we are unable to deliver a message after this time period, we convert these deferrals to "Hard Bounces". When an address is hard bounced either due to this, or because we got a different class of error (such as "this user does not exist"), we add this address to the Block list.
-
-What this means is that even if you send a message through us for that user, we will not even try to deliver to that address, because we know it's no longer good. Continuing to try to send to a known bad address will harm your reputation with the receiver, so we prevent that. However, if you continue to try to send, and we intercept it, your reputation with us goes down, as noted on your [Dashboard]({{site.app_url}}).
-
-{% anchor h2 %}
-Recap
-{% endanchor %}
+-   Some recipients have marked your mail as spam, but not enough for the server to block you. Instead, they will refuse to receive more until they can learn more about how their users intereact with your messages.
 
-Deferrals, aka Throttling, aka Soft Bounces are a normal part of email delivery, one SendGrid knows how to handle. There are a number of mundane reasons that can cause a defer. If a message stays throttled, it will be added to the Block list.
+If a message is continuously deferred for 72 hours, it may mean that there is another issue such as the account being inactive, a full mailbox, or some other legitimate problem with the recipient. When we are unable to deliver a message after this time period, we convert these deferrals to "Blocks". Blocks are simply stored for reference, and do not behave like a supression list when sending to addresses that are on the list. 
```

Comment:
~~Blocks are simply stored for reference, and do not behave like a supression list when sending to addresses that are on the list. ~~

Blocks are stored for reference and do not behave like a suppression list when sending to addresses that are on the list.